aptos-move/framework/aptos-framework/sources/delegation_pool.move

```diff
@@ -1412,6 +1506,17 @@ module aptos_framework::delegation_pool {
         // in order to appreciate all shares on the active pool atomically
         buy_in_active_shares(pool, NULL_SHAREHOLDER, add_stake_fee);
 
+        if (features::module_event_migration_enabled()) {
+            event::emit(
+                AddStakeEvent {
+                    pool_address,
+                    delegator_address,
+                    amount_added: amount,
+                    add_stake_fee,
+                },
+            );
+        };
+
         event::emit_event(
```

Comment:
Do we need to emit both events after migration?

Reply:
We delete v1 afterwards.

Reply:
Afterwards meaning next release? Does it make sense to make old logic dependent on the flag not set so we won't have a period where we fire two events at the same time?

Reply:
we must have a period that both exist. like rolling update....